### PR TITLE
Fix/3997  csp collabora

### PR DESCRIPTION
**Pull request by grignards**
See #3997.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by raphj — APPROVED** [review]
Don't forget to document usage of the settings for additional directives if relevant

File: backend/tracim_backend/views/frontend.py [hunk 1]
```diff
@@ -95,8 +95,21 @@ def index(self, context, request: TracimRequest):
...
+            app_lib = ApplicationApi(app_list=app_list)
+            for app in app_lib.get_all():
+                app_directives = app.get_content_security_policy_directives(app_config)
+                for app_key, app_value in app_directives:
+                    try:
+                        csp_directives[app_key] = "{} {}".format(csp_directives[app_key], app_value)
+                    except KeyError:
+                        csp_directives[app_key] = app_value
+
+            csp = "; ".join("{} {}".format(k, v) for k, v in csp_directives.items())
```
Comment (inkhey) [inline backend/tracim_backend/views/frontend.py:111]: not really fan of single character variable but for this usage it's probably acceptable.

Comment (raphj) [reply backend/tracim_backend/views/frontend.py:111]: k, v for key, value in a comprehension is ok for me indeed :-)

**Comment by grignards** [comment]
> Don't forget to document usage of the settings for additional directives if relevant

No changes to the configuration in this PR.

**Comment by grignards** [comment]
Tested on branch with a local collabora.

**Pull request merged**